### PR TITLE
Use go modules instead of dep, easier development experience (requires go 1.11)

### PR DESCRIPTION
**Pull request by Gabrn**
- No need for $GOPATH
- Use module name (tweek-gateway) instead of git repo name for project own packages.

### CODE REVIEW

**Comment by Gabrn** [comment]
Weird that it doesn't pass on code-fresh. 
`yarn start:e2e` seems to pass locally

File: services/gateway/appConfig/appConfig.go [hunk 1]
```diff
@@ -103,34 +103,38 @@ type PolicyStorage struct {
...
 	tweekConfigor := configor.New(&configor.Config{ENVPrefix: "TWEEKGATEWAY"})
-
-	configFilePath := "/settings/settings.json"
-	if _, err := os.Stat(configFilePath); !os.IsNotExist(err) {
-		if err = tweekConfigor.Load(conf, configFilePath); err != nil {
-			logrus.WithError(err).Error("Configuration error")
+	const settingsFilePath = "./settings/settings.json"
+	if _, err := os.Stat(settingsFilePath); !os.IsNotExist(err) {
+		if err = tweekConfigor.Load(conf, settingsFilePath); err != nil {
+			logrus.WithError(err).Error("Settings error", err)
```
Comment (michaelkruglos) [inline services/gateway/appConfig/appConfig.go]: no need to have `, err)` there. It's already reported in `WithError`

File: services/gateway/appConfig/appConfig.go [hunk 2]
```diff
@@ -103,34 +103,38 @@ type PolicyStorage struct {
...
 		}
 	} else {
-		logrus.WithField("error", err).Panic("Config file not found")
+		logrus.WithField("error", err).Panic("Settings file not found:", err)
```
Comment (michaelkruglos) [inline services/gateway/appConfig/appConfig.go]: Same as above.

**Pull request merged**